crates/net/network/src/fetch/mod.rs

```diff
@@ -234,6 +234,11 @@ impl StateFetcher {
             status_tx: self.status_tx.clone(),
         }
     }
+
+    /// Returns sender half of the [`StatusUpdate`] channel.
+    pub(crate) fn update_status_tx(&self) -> UnboundedSender<StatusUpdate> {
+        self.status_tx.clone()
+    }
```

Comment:
I think this no longer needed here, so we can remove this channel entirely and instead use the `handle -> manager` channel and add StatuUpdate as a variant there.

Reply:
If I understand correctly, move the channel to `NetworkState` and have that listen to `StatusUpdate` messages through the `NetworkHandle`?

Reply:
Scrap that last comment, should be fixed now